api/src/opentrons/config/reset.py

```diff
@@ -47,6 +48,10 @@ class ResetOptionId(str, Enum):
         name="Tip Length Calibrations",
         description="Clear tip length calibrations (will also clear " "pipette offset)",
     ),
+    ResetOptionId.db_history: CommonResetOption(
+        name="Clear Runs History",
+        description="Erase this device's stored history of protocols and runs.",
+    ),
```

Comment:
We discussed this in a couple of our calls and I think this is fine for this PR, but maybe let's leave a `# TODO` for giving this setting a proper home?

```suggestion
    # TODO(mm, 2022-05-23): Run and protocol history is a robot-server thing,
    # and is not a concept known to this package (the `opentrons` library).
    # This option is defined here only as a convenience for robot-server.
    # Find a way to split thing up and define this in robot-server instead.
    ResetOptionId.db_history: CommonResetOption(
        name="Clear Runs History",
        description="Erase this device's stored history of protocols and runs.",
    ),
```